docs/core/concepts/execution.md

```diff
@@ -81,7 +81,33 @@ Another common use of Prefect signals is when the task in question will be neste
 
 Prefect provides a powerful `Context` object to share information without requiring explicit arguments on a task's `run()` method.
 
-The `Context` can be accessed at any time, and will be pre-populated with information before and during each flow run. For an exhaustive list of values that you can find in context, see the corresponding [API documentation](../../api/latest/utilities/context.html).
+The `Context` can be accessed at any time, and Prefect will populate it with information during flow and task execution. The context object itself can be populated with any arbitrary user-defined key-value pair, which in turn can be accessed with dot notation.
+
+### Adding context globally
+Adding context globally is possible via your `config.toml` in a section called `[context]`. For any keys specified here, as long as Prefect Core does not override this key internally, it will be accessible globally from `prefect.context`, even outside of a flow run.
+```
+# config.toml
+[context]
+a = 1
+```
+
+```python
+>>> import prefect
+>>> prefect.context.to_dict()['a']
```

Comment:
I believe we can remove the need to call `to_dict` since context acts in a similar manner to a dictionary already!

```python
In [1]: from prefect import context

In [2]: context.test
Out[2]: 'here'

In [3]: context.get('test')
Out[3]: 'here'

In [4]: context['test']
Out[4]: 'here'
```

Reply:
Duh u right! Updated in https://github.com/PrefectHQ/prefect/pull/2110/commits/69e697b31274215c92e0a1affd8de6691a592f45 with some extra examples off of your comment here